### PR TITLE
Fix api to be ocs

### PR DESCRIPTION
**Pull request by jotoeri**
Now getting a true ocs response. Unfortunately this produces a deep object concattenation, which is why i use a small wrapper to extract the actual data.

### CODE REVIEW

File: lib/Controller/ApiController.php [hunk 1]
```diff
@@ -123,6 +123,7 @@ public function __construct(string $appName,
 
 	/**
 	 * @NoAdminRequired
+	 * @NoCSRFRequired
```
Comment (jotoeri) [inline lib/Controller/ApiController.php:126]: @nickvergessen , if i remove this here, i get a CSRF Error.
Is there a better way to solve this then?

Comment (jotoeri) [reply lib/Controller/ApiController.php:126]: ah... wait will check if i maybe missed a header. :(

Comment (jotoeri) [reply lib/Controller/ApiController.php:126]: Ok, sorry. My fault.
Good we talked about, thank you! 😉

Comment (nickvergessen) [reply lib/Controller/ApiController.php]: 🦆  Rubber duck to the rescue

**Comment by jotoeri** [comment]
Splitted this, as it became too much. Now it's only what is said in the title. 😉

**Comment by jotoeri** [comment]
@loelkes might want to give here a review, too? :)

**Comment by loelkes** [comment]
@jotoeri Awesome! I can't look right now, maybe next week?

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**